pkgs/by-name/pr/prism-model-checker-unwrapped/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  coreutils,
+  fetchFromGitHub,
+  java,
+  cctools,
+}:
+stdenv.mkDerivation rec {
```

Comment:
rec isn't ideal for use with `stdenv.mkDerivation`. If you can adapt this to use the functional `finalAttrs` model that'd be good :slightly_smiling_face: